src/store-tree/index.ls

```diff
@@ -71,6 +71,12 @@ class StoreTree extends StoreNode
     obj
 
 
+  $debug: ->
+    obj = {}
+    @_for-each-subnode (subnode, key) -> obj[key] = subnode.$debug!
```

Comment:
We could probably overwrite `@_children[key]` in the constructor to also point to the same object which can remove this helper though